apps/hash-external-services/docker-compose.test.yml

```diff
@@ -17,8 +17,46 @@ services:
       HASH_GRAPH_PG_DATABASE: "${HASH_GRAPH_PG_TEST_DATABASE}"
 
   graph-test-server:
+    init: true
+    depends_on:
+      postgres:
+        condition: service_healthy
+      graph-migrate:
+        condition: service_completed_successfully
+    image: hash-graph
+    read_only: true
+    security_opt:
+      - no-new-privileges:true
+    volumes:
+      - log:/log
```

Comment:
Is the `log` volume found properly when we move it out like this?

Reply:
Yes, I just checked it again:
<img width="578" alt="image" src="https://user-images.githubusercontent.com/21277928/237046975-e12c775e-1a13-44da-b799-f01f108b8492.png">